src/assets/scripts/game/GameController.js

```diff
@@ -1,13 +1,35 @@
 /* eslint-disable camelcase, no-underscore-dangle, no-mixed-operators, func-names, object-shorthand,
 no-undef, class-methods-use-this */
 import $ from 'jquery';
+import _forEach from 'lodash/forEach';
 import GameOptions from './GameOptions';
 import { round } from '../math/core';
 import { SELECTORS } from '../constants/selectors';
+import { TIME } from '../constants/globalConstants';
 
 // Temporary const declaration here to attach to the window AND use as internal property
 const game = {};
 
+// TODO: Remember to move me to wherever the constants end up being moved to
+/**
+ * Definitions of point values for given score events
+ * @type {Object}
+ */
+export const GAME_EVENTS = {
```

Comment:
I _love_ how you've centralized these constants and I especially love how you've added the TODO at the top.  I was about to ask if this is where these should go, and I can see you're already thinking ahead.  Great job!